docs/structure.md

```diff
@@ -46,4 +46,4 @@ Once you create your own strategy or use a ready strategy from the [pyalgostrate
 
 ## What's Next...
 ---
-Next, you code the actual strategy taken as an example in this tutorial. Alternatively, you can pick up a ready strategy and directly [upload](upload.md) it.
\ No newline at end of file
+Next, you code the actual strategy taken as an example in this tutorial. Alternatively, you can pick up a ready strategy and directly [upload](upload.md) it.
```

Comment:
Remove the newline at the end of the file.

Reply:
I have updated the changes.